packages/vaadin-lumo-styles/font-icons.js

```diff
@@ -4,63 +4,62 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import './version.js';
+import { css } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { addLumoGlobalStyles } from './global.js';
```

Comment:
These changes need to be added to `gulpfile.js` in order to be retained on re-generating. I will work on it.